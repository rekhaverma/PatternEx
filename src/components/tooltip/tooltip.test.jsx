import React from 'react';
import Tooltip from './tooltip.component';

describe('<Tooltip />', () => {
  let wrapper;
  let props;

  beforeEach(() => {
    props = {
      trigger: (<div />),
      children: (<div />),
      managerStyle: {},
      position: 'top',
      hideWhenHoveringTooltipContent: false,
      style: {},
    };
    wrapper = shallow(<Tooltip {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

