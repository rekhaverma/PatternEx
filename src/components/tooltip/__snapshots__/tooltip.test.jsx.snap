// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tooltip /> should render correctly 1`] = `
<div
  className="tooltip"
>
  <Manager
    style={Object {}}
    tag="div"
  >
    <Target
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div />
    </Target>
  </Manager>
</div>
`;
